.github/workflows/publish_code.yml

```diff
@@ -1,6 +1,7 @@
 name: Publish release
 
 on:
+  workflow_dispatch:
```

Comment:
I am adding this to enable us to manually trigger this job. Normally it will automatically run whenever a tag gets pushed so manual intervention should really only be needed in cases where the action fails (like this time).